config/jobs/image-pushing/k8s-staging-cluster-api.yaml

```diff
@@ -6,6 +6,9 @@ postsubmits:
       annotations:
         # this is the name of some testgrid dashboard to report to.
         testgrid-dashboards: sig-cluster-lifecycle-image-pushes, sig-k8s-infra-gcb
+        testgrid-tab-name: post-cluster-api-push-images
```

Comment:
This is already the current tab name. Just thought I'd make it explicit while I'm here